### ISSUE TITLE
Allow creatives to be scheduled

### ISSUE THREAD

**Issue by sumobi** [AffiliateWP/AffiliateWP#982]
Allow creatives to be scheduled, this would be helpful all year round. 

An admin can add a bunch of creatives for each holiday/special occasion, and once those dates come into effect, the creatives are shown in the affiliate area automatically. They're then hidden once the date has passed.

**Comment by pippinsplugins** [comment]
This would be awesoe.

**Comment by sumobi** [comment]
Yep, if you can schedule discount codes, MailChimp emails and creatives, you could automate sales all year round :+1:

**Comment by michaelbeil** [comment]
Oh my :100:.

**Comment by michaelbeil** [comment]
A user has requested this here: https://secure.helpscout.net/conversation/182566212/30213/?folderId=634609
